### PR TITLE
User Story 34 - Merchant Dashboard Show Page

### LINKED ISSUE #5: User Story 34, Merchant Dashboard Show Page
As a merchant employee or admin
When I visit my merchant dashboard ("/merchant")
I see the name and full address of the merchant I work for

### PR DESCRIPTION
**Pull request by jfangonilo**
## What functionality does this accomplish?
closes #5 

**Description:**
Adds functionality for an optional user belongs_to merchant relationship. Merchant info now appears on merchant dashboard page. Factories updated to allow for this functionality

### Test Coverage Percentage: 98.21%
- [ ] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
- [x] New Tests added for new features

## Checklist:
- [x] My code has no unused/commented out code
- [ ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Please include a gif of how you feel about this branch:
chugging kramer

### CODE REVIEW

File: spec/features/merchant/dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant_employee)
+
+    visit "/merchant"
+
+    expect(page).to have_content merchant_employee.name
+    expect(page).to have_content merchant_employee.address
+    expect(page).to have_content merchant_employee.city
+    expect(page).to have_content merchant_employee.state
+    expect(page).to have_content merchant_employee.zip
```
Comment (itemniner) [inline spec/features/merchant/dashboard_spec.rb]: Looks great! Im not sure if I read the story wrong but i think 34 wants the employers?

Comment (itemniner) [reply spec/features/merchant/dashboard_spec.rb]: employer information*

Comment (jfangonilo) [reply spec/features/merchant/dashboard_spec.rb]: Oh yep, you're right! `merchant_employee` is a User class. I'd need to pull the merchant info. Good catch

Comment (jfangonilo) [reply spec/features/merchant/dashboard_spec.rb]: Made the changes and pushed them up. Let me know how it looks now

Comment (wrenisit) [reply spec/features/merchant/dashboard_spec.rb]: Great Job!

**Pull request merged**